@abstractZono/abstractZono.m

```diff
@@ -77,6 +71,16 @@
             bb = obj.boundingBox;
             out = bb.c - sum(abs(bb.G),2);
         end
+        function [lb,ub] = bounds(obj)
+            bb = obj.boundingBox;
+            ub = bb.c + sum(abs(bb.G),2);
+            lb = bb.c - sum(abs(bb.G),2);
+        end
+    end
+
+    %% Additional Overide Methods
+    methods
+        function out = linMap(in,M); out = mtimes(M,in); end
```

Comment:
How does this syntax even work?  This defines linMap and mtimes?  as the same thing?  Are these changes needed?  I'd like to get buy in from Dr. Koeln if we are going to make changes to the main library....

Reply:
All this does is switch the order calling mtimes() so that it will call the correct method (needed for optimvar type stuff).